frontend-react/src/contexts/SessionStorageTools.ts

```diff
@@ -0,0 +1,77 @@
+/* I put this in the `/contexts` directory because it felt
+ * right, but by no means is it a context. Feel free to move
+ * it if you can think of a better spot. */
+import { PERMISSIONS } from "../resources/PermissionsResource";
+import { groupToOrg } from "../webreceiver-utils";
+import { SessionStore } from "../hooks/UseSessionStorage";
+
+export enum GLOBAL_STORAGE_KEYS {
+    GLOBAL_BASE = "global-context-",
+    GLOBAL_ORG = "global-context-org",
+    SENDER_MODE = "global-sender-mode",
+    SENDER_NAME = "global-sender-name",
+    OKTA_ACCESS_TOKEN = "global-okta-token",
+}
+
+export function clearGlobalContext(): void {
+    for (let key in sessionStorage) {
+        sessionStorage.removeItem(key);
+    }
+}
+
+export function parseOrgs(orgs: Array<string>): Array<Partial<SessionStore>> {
+    return orgs.map((org) => {
+        if (org.includes(PERMISSIONS.SENDER)) {
+            const sender = org.split(".");
+            return {
+                org: groupToOrg(sender[0]),
+                senderName: sender[1] || "default",
+            };
+        } else {
+            return {
+                org: groupToOrg(org),
+                senderName: undefined,
+            };
+        }
+    });
+}
+
+export function storeParsedOrg(parsedVal: {
+    org: string;
+    sender: string | undefined;
+}) {
+    // Sets az-phd as default when Admins have no state or sender orgs
+    if (parsedVal.org.includes("PrimeAdmin")) {
+        setStoredOrg("az-phd");
```

Comment:
This can be altered; this was intended to set the default to `az-phd` for admins but A) it gets overwritten by another check later, and B) I don't want to put any org names other than `PrimeAdmins` and `ignore` in the code if I can help it.

Reply:
Maybe load it from the `.env.` file? Or at least put it in a constant so it stands out? 
`DEFAULT_ADMIN_ORG_ON_LOGIN` is pretty long but understandable?